foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/TestAddressResoverConfig.java

```diff
@@ -40,8 +61,7 @@ public void testGetResover() {
     ArchaiusUtils.setProperty("addressResolver.ndots", 3);
     ArchaiusUtils.setProperty("addressResolver.rotateServers", true);
     AddressResolverOptions aroc = AddressResolverConfig.getAddressResover("test");
-    Assert.assertEquals(2, aroc.getServers().size());
-    Assert.assertEquals("8.8.8.8", aroc.getServers().get(0));
+    Assert.assertThat(aroc.getServers(), is(Arrays.asList("8.8.8.8", "8.8.4.4")));
     Assert.assertEquals("test.svc.local.cluster", aroc.getSearchDomains().get(0));
```

Comment:
This line can be changed to use assertThat.

Reply:
ok